howto_resize_image.md

```diff
@@ -0,0 +1,14 @@
+# How to resize image
+
+To make an image resize, we are using the method `load` of the class [Image](https://reference.aspose.com/imaging/net/aspose.imaging/image/) to load an image. Then we [resize](https://reference.aspose.com/imaging/net/aspose.imaging/image/resize/) the image to a new size by specifying desired height and width `300x300` in pixels and save the result to a new file. 
+
+Example C# code:
+
+{% gist aspose-com-gists/7ee37a401e37790396ad9f4cde87d446 simple-resizing-image.cs %}
+
+Additionally, you could resize proportionally the image width or height and specify the resizing type by selecting `ResizeType.LanczosResample` parameter:
```

Comment:
@anms23 , how do you think maybe each described operation should contain example of results?

Reply:
I had thought some operations such as Resize, Crop and Rotate are obvious and may not need image examples for demonstrations, but I could add more examples.

Reply:
@anms23 , ok thanks, I can not say you is it needed or not, in my opinion when page contains graphics it is more attractive. I'll discuss it with Alexander, what he thinks about it.

Reply:
@anms23 , Alexander thinks that it is possible, but he reminded me that we should support alt tag for images for SEO purposes. Can you please also update it?

Reply:
@sam199008 , Updated:
https://github.com/aspose-imaging/aspose-imaging-probe/pull/2/commits/efa302e8c9fb1f520248dd7b1aa89deb2534afc3